client/controller/index.web.js

```diff
@@ -219,6 +224,25 @@ export function redirectWithoutLocaleParamIfLoggedIn( context, next ) {
 	next();
 }
 
+/**
+ * Removes the locale parameter from the beginning of the path, and redirects logged-in users to it.
+ *
+ * @param   {Object}   context Context object
+ * @param   {Function} next    Calls next middleware
+ * @returns {void}
+ */
+export const redirectWithoutLocaleParamInFrontIfLoggedIn = ( context, next ) => {
```

Comment:
```suggestion
export const redirectWithoutLocaleParamIfLoggedIn = ( context, next ) => {
```
I like the renaming, but I think the "in front" part is kind of redundant. I'll leave the decision to you.

Reply:
Well that's on purpose, because there is a method named `redirectWithoutLocaleParamIfLoggedIn()` just above. I was hoping that this would help distinguish them 🙂.

Reply:
Oh! It lives right above this function, yet I overlooked it. Knowing that we also have `redirectWithoutLocaleParamIfLoggedIn`, I think it makes even more sense to rename this function